### DISCUSSION TITLE
How many chia forks are you farming

### DISCUSSION THREAD

**Discussion by dedezagaha** [raingggg/coctohug#2] [General]
I have a regular PC, and not sure how many forks should I config. Any ideas?

**Comment by jamisshow** [comment]
Just if your pc is more powerful than intel i5 cpu level, you should be able to farm more than 5.
However, memory might be a bottle neck for you. As their doc mentioned, one fork might use 1.8G ram. 

Here is the hardware link: https://github.com/raingggg/coctohug/blob/main/docs/wiki/wiki_en.md#cch-hardware_requirements

**Comment by sirthiswe** [comment]
My RAM is 64G, so I can mine 30+ forks
